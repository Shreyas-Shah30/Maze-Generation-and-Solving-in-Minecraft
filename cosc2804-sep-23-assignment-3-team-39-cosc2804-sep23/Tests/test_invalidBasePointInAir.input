1
1
4028 73 4241
4017 68 4237
7 5
x.xxx
x.x.x
x.x.x
x.x.x
x.x.x
x...x
xxxxx
2
6
