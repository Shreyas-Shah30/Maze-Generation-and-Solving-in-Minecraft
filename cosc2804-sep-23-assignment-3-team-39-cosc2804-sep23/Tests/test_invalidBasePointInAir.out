
Welcome to MineCraft MazeRunner!
--------------------------------

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 

------------- GENERATE MAZE -------------
1) Read Maze from terminal
2) Generate Random Maze
3) Back

Enter Menu item to continue: 
Enter the basePoint of the maze:
Input Error: The coordinates entered are in the air. Please enter coordinates exactly one unit above the ground.
Enter the length and width of the maze:
Enter the maze structure:
Maze read successfully
**Printing Maze**
BasePoint: (4017, 68, 4237)
Structure:
x.xxx
x.x.x
x.x.x
x.x.x
x.x.x
x...x
xxxxx
**End Printing Maze**

------------- MAIN MENU -------------
1) Generate Maze
2) Build Maze in MineCraft
3) Solve Maze
4) Enhancements Menu
5) Show Team Information
6) Exit

Enter Menu item to continue: 
The End!

